wolfcrypt/src/random.c

```diff
@@ -647,14 +647,7 @@ static int Hash_DRBG_Generate(DRBG_internal* drbg, byte* out, word32 outSz)
         return DRBG_NEED_RESEED;
     }
     else {
-    #ifndef WOLFSSL_SMALL_STACK
```

Comment:
This change isn't going into the FIPS build.
Is the customer using FIPS?

Reply:
Yes but that's OK -- this change is just for future benefits.